lib/web_ui/lib/src/engine/canvaskit/image_web_codecs.dart

```diff
@@ -72,7 +179,28 @@ class CkBrowserImageDecoder implements ui.Codec {
       // package:js bindings don't work with getters that return a Promise, which
       // is why js_util is used instead.
       await promiseToFuture<void>(getJsProperty(webDecoder, 'completed'));
-      return CkBrowserImageDecoder._(webDecoder, debugSource);
+      frameCount = webDecoder.tracks.selectedTrack!.frameCount;
+      repetitionCount = webDecoder.tracks.selectedTrack!.repetitionCount;
+
+      _cachedWebDecoder = webDecoder;
+
+      // Expire the decoder if it not used for several seconds. If the image is
```

Comment:
Typo: `// Expire the decoder if -it's- not used for several seconds. If the image is`

Reply:
Done.